.github/release-drafter.yml

```diff
@@ -14,6 +16,8 @@ categories:
     labels:
       - 'bug-py-minor'
       - 'bug-py'
+  - title: 'Dialects'
```

Comment:
Should we remove the `Compiler` title?
AFAIK, we renamed `Compiler` to `Dialects`?